js/index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var toggleSidebar = function(){
+   // var sidebarSmallerScreens = document.querySelector('#sidebar-smaller-screens');
+    var sidebarMobile = document.querySelector('#sidebar-mobile')
+
+   // sidebarSmallerScreens.classList.toggle('sidebar-mobile', 'sidebar-smaller-screens');
+    sidebarMobile.classList.toggle('sidebar-smaller-screens');
```

Comment:
chowasz mniejsze menu i to jest ok, ale musisz rownoczesnie pokazac to wieksze - jest jakas klasa, ktora to u Ciebie zapewni?